From 7a2064b23480d12beabfbfa67fdd9a4019179edc Mon Sep 17 00:00:00 2001
From: "Tan, Raymond" <raymond.tan@intel.com>
Date: Thu, 14 May 2015 14:16:21 +0800
Subject: [PATCH 282/389] gpio-sch: fix ack() and switch to std IRQ flow
 handler

This patch is ported from Quark BSP R1.1, with the same name,
for RTC #75539 #78738

Original author is Josef Ahmad <josef.ahmad@intel.com>

Signed-off-by: Tan, Raymond <raymond.tan@intel.com>
Change-Id: Ieb81aa30bc9b0975999e4ff2649ebc79e6ea842c
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/4990
---
 drivers/gpio/gpio-sch.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpio/gpio-sch.c b/drivers/gpio/gpio-sch.c
index 48b03e3..2d2c1f1 100644
--- a/drivers/gpio/gpio-sch.c
+++ b/drivers/gpio/gpio-sch.c
@@ -282,7 +282,7 @@ static void sch_gpio_irqs_init(struct sch_gpio *sch, unsigned int num)
 	for (i = 0; i < num; i++) {
 		irq_set_chip_data(i + sch->irq_base, sch);
 		irq_set_chip_and_handler_name(i + sch->irq_base, &sch_irq_chip,
-					handle_simple_irq, "sch_gpio_irq_chip");
+					handle_edge_irq, "sch_gpio_irq_chip");
 	}
 }
 
-- 
1.9.3

